eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/NodeRelayer.scala

```diff
@@ -257,15 +256,25 @@ object NodeRelayer {
    * This helper method translates relaying errors (returned by the downstream nodes) to a BOLT 4 standard error that we
    * should return upstream.
    */
-  private def translateError(failures: Seq[PaymentFailure], outgoingNodeId: PublicKey): Option[FailureMessage] = {
+  private def translateError(nodeParams: NodeParams, failures: Seq[PaymentFailure], p: PendingResult): Option[FailureMessage] = {
```

Comment:
One more occurence (see https://github.com/ACINQ/eclair/pull/1361#issue-396455215) in favor of traits that would be implemented by `NodeParams`:

```scala

sealed trait RelayFeeConf {
  def feeBase: MilliSatoshi
  def feeProportionalMillionth: Int
}

case class NodeParams extends RelayFeeConf {
...
}
```

```suggestion
  private def translateError(relayFeeConf: RelayFeeConf, failures: Seq[PaymentFailure], p: PendingResult): Option[FailureMessage] = {
```

Probably not worth it here since it's kind of a hack, but still. It's annoying to have to carry the whole `NodeParams` just for two parameters.